### DISCUSSION TITLE
TacticalRMM Update-Error-monitor type

### DISCUSSION THREAD

**Discussion by Zach-Harald** [amidaware/tacticalrmm#2059] [Q&A]
Hello,
after updating tacticalrmm to the new version, no servers or workstations are displayed in the frontend, and the version number is no longer displayed. I get the error message "500 internal server error". The clients and sites are in the organization tree. When I click on a site, I get the error message "400 bad request, monitoring type does not exist". The menu works, I can create clients or new scripts.

How can the error be fixed?

**Comment by wh1te909** [comment]
what's the actual error message from the logs? https://docs.tacticalrmm.com/troubleshooting/#error-500

**Comment by Zach-Harald** [comment]
The last line of the log is: redis.exceptions.connectionError: Error 111 connecting to 127.0.0.1:6379. Connection refused.

 



 

 

Mit freundlichen Grüßen

 

Dipl.-Ing. (FH)

Harald Zach

Nadistr. 20

80809 München

 

Telefon : (089) 99 02 03 40

Fax        : (089) 99 02 03 42

Mobil     : (0171) 9 38 19 36

Internet :  <https://it-tataa.de/> https://zach1.de

 

Von: wh1te909 ***@***.***> 
Gesendet: Samstag, 2. November 2024 01:41
An: amidaware/tacticalrmm ***@***.***>
Cc: Zach-Harald ***@***.***>; Author ***@***.***>
Betreff: Re: [amidaware/tacticalrmm] TacticalRMM Update-Error-monitor type (Discussion #2059)

 

what's the actual error message from the logs? https://docs.tacticalrmm.com/troubleshooting/#error-500

**Comment by bettoinformatica** [comment]
I have a similar problem. After upgrading to version 0.19.4 from version 0.19.3, when I try to connect it gives me "internal server error 500". During the installation it gives me a series of errors. Where can I find the log file and who should I send it to?
I have now reverted to the previous version with the backup I made.
Thank you in advance for your help.

**Reply by wh1te909** [reply]
check my first reply to this thread, i already pasted the docs link that tells you what file to find the exact error in

**Comment by Zach-Harald** [comment]
I checked with "systemctl --failed" any failed services on my machine.

I got the following:

<img width="790" alt="image" src="https://github.com/user-attachments/assets/5adf545f-d456-486f-be71-8593e99563e6">

How can I fix this?

Thankyou for help.

**Reply by dinger1986** [reply]
why do you have vsftpd on your server?

**Reply by Zach-Harald** [reply]
for downloading files to my pc. For example the backup-file for restoring to another machine.

**Reply by dinger1986** [reply]
what other services are installed?

**Reply by Zach-Harald** [reply]
All services that the installation of tacticalrmm provides. vsftpd is the only service installed subsequently. TacticalRMM worked perfectly until the update to the new version

**Reply by dinger1986** [reply]
ok, what errors did you get? Can you dm me them on discord?

**Reply by Zach-Harald** [reply]
When I start TRMM, I can log in with username and password. After I enter the 2FA token, the TRMM overview opens.
The red error box with "500: Internal Server Error" then appears at the top of the screen. When clicking on a site, the error box "monitoring type does not exist, 400: Bad Request" appears. No version is displayed at the top left, only a small v is visible. When I open my preferences I get the error message "500: Internal Server Error" and the form is completely empty.

<img width="1178" alt="image" src="https://github.com/user-attachments/assets/8201f076-c556-4828-8737-442e3470b453">

**Reply by wh1te909** [reply]
you already found and pasted the error last week when I asked you to check the log: `The last line of the log is: redis.exceptions.connectionError: Error 111 connecting to 127.0.0.1:6379. Connection refused.`

you can search online for "troubleshoot redis not running" fix redis and then run the update again
